cmd/database.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"xkcdcomics/model"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var Db *sql.DB
+
+func SetupDatabase() (*sql.DB, error) {
+	Db, err := sql.Open("sqlite3", "./xkcd_comics.db")
+	if err != nil {
+		return nil, fmt.Errorf("failed to open database: %v", err)
+	}
+
+	// Create table if not exists
+	_, err = Db.Exec(`CREATE TABLE IF NOT EXISTS comics (
+		num INTEGER PRIMARY KEY,
+		safeTitle TEXT,
+		img TEXT,
+		title TEXT,
+		transcript TEXT
+	)`)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create table: %v", err)
+	}
+
+	return Db, nil
+}
+
+func InsertComic(db *sql.DB, c model.Comic) error {
+	_, err := db.Exec("INSERT INTO comics (num, safeTitle, img, title, transcript) VALUES (?, ?, ?, ?, ?)",
```

Comment:
Since we get them in batches anyway, why not insert batches in the database too? Something like [this](https://stackoverflow.com/questions/21108084/how-to-insert-multiple-data-at-once). This way we maximize the number of operations we can do per second by running fewer individual transactions